projects/extension/sql/idempotent/011-grant-to.sql

```diff
@@ -0,0 +1,13 @@
+-------------------------------------------------------------------------------
+-- grant_to
+create or replace function ai.grant_to(variadic grantees name[]) returns name[]
+as $func$
+    select pg_catalog.array_agg(cast(x as name))
+    from (
+        select pg_catalog.unnest(grantees) x
+        union
+        select pg_catalog.unnest(pg_catalog.string_to_array(pg_catalog.current_setting('ai.grant_to_default', true), ',')) x
```

Comment:
<img width="616" alt="image" src="https://github.com/user-attachments/assets/b2f79e63-ec0c-4698-95b7-954771372f41">
Having a username with a comma in it is unlikely but possible.

Reply:
not worth the effort